experimenter/tests/integration/nimbus/android/conftest.py

```diff
@@ -0,0 +1,154 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+import os
+import subprocess
+import time
+from pathlib import Path
+
+import pytest
+import requests
+
+from .gradlewbuild import GradlewBuild
+from .models.models import TelemetryModel
+
+here = Path().cwd()
+
+
+def start_process(path, command):
+    module_path = Path(path)
+
+    try:
+        process = subprocess.Popen(
+            command,
+            encoding="utf-8",
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            universal_newlines=True,
+            cwd=module_path.absolute(),
+        )
+        stdout, stderr = process.communicate(timeout=5)
+
+        if process.returncode != 0:
+            raise Exception(stderr)
```

Comment:
we seem to be throwing away the IO, so we should use `subprocess.check_call()` with `stdout` and `stderr` set to `subprocess.DEVNULL`

Reply:
Would this block the python script from continuing? The ping server just needs to start, I remember trying `check_output` and I couldn't use it because it was blocking and the server has to stay running, but I couldn't start the test suite.